frontend/src/pages/secret-manager/IntegrationsListPage/route.tsx

```diff
@@ -35,10 +52,21 @@ export const Route = createFileRoute(
         });
       }
 
-      const integrations = await context.queryClient.ensureQueryData({
-        queryKey: workspaceKeys.getWorkspaceIntegrations(projectId),
-        queryFn: () => fetchWorkspaceIntegrations(projectId)
-      });
+      let integrations: TIntegration[];
+      try {
+        integrations = await context.queryClient.ensureQueryData({
```

Comment:
Follow this pattern later - when we re-organization react query keys to be https://tkdodo.eu/blog/the-query-options-api

This was not done on migration on tanstack router migration due to scope and time limitation.